### PR TITLE
feat(deployment): Add

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR adds/fixes ...

**Review:**
- [ ] Tests executed by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
Together with: https://github.com/Clinical-Genomics/servers/pull/1436

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+` ### Fixed `
+
+6. Review the details once again and merge the branch into master.
+
+
+7. Wait for GitHub actions to process the event, bump version, create release, publish to Dockerhub and PyPi.
+
+
+8. Deploy master to stage:
+    1. Log in to appropriate server `ssh <server.scilifelab.se>`
```
Comment (ChrOertlin) [inline DEPLOYMENT.md:33]: can we make this explicit as to what server is the appropriate one?

File: DEPLOYMENT.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+       bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_crunchy -t crunchy -b master
+       ```
+   5. ```shell
+      bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-crunchy-containers-stage.sh -e S_crunchy -t crunchy -b master
+      ```
+   6. Make sure that installation was successful
+   
+
+9. Deploy master to production
+     1. Log in to appropriate server `ssh <server.scilifelab.se>`
```
Comment (ChrOertlin) [inline DEPLOYMENT.md:46]: same here

**Review by ChrOertlin — APPROVED** [review]
👍

File: DEPLOYMENT.md (hunk 2 above)
Comment (henrikstranneheim) [reply DEPLOYMENT.md]: Fixed

File: DEPLOYMENT.md (hunk 1 above)
Comment (henrikstranneheim) [reply DEPLOYMENT.md]: Fixed

**Comment by henrikstranneheim** [comment]
```
[henrik.stranneheim@hasta:~] [P_base] $ bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-prod.sh -e P_crunchy -t crunchy -b master
Collecting git+https://github.com/Clinical-Genomics/crunchy@master
  Cloning https://github.com/Clinical-Genomics/crunchy (to revision master) to ./tmp/pip-req-build-8ndlu9nl
  Running command git clone --quiet https://github.com/Clinical-Genomics/crunchy /home/henrik.stranneheim/tmp/pip-req-build-8ndlu9nl
  Resolved https://github.com/Clinical-Genomics/crunchy to commit a82b32b6818b1b1a1dd392b29c1924c9ae8507d6
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: crunchy
  Building wheel for crunchy (setup.py) ... done
  Created wheel for crunchy: filename=crunchy-1.0.5-py3-none-any.whl size=18134 sha256=8b99fda48a1c739527c326504672e222eb31d48a5e584c4880b9010a511177c2
  Stored in directory: /home/henrik.stranneheim/tmp/pip-ephem-wheel-cache-afil63bl/wheels/f4/49/06/6699608ba656b7a332be1a2a9ea5d2794ba953775ffad39ac4
Successfully built crunchy
Installing collected packages: crunchy
  Attempting uninstall: crunchy
    Found existing installation: crunchy 1.0.2
    Uninstalling crunchy-1.0.2:
      Successfully uninstalled crunchy-1.0.2
Successfully installed crunchy-1.0.5
Collecting git+https://github.com/Clinical-Genomics/crunchy@master
  Cloning https://github.com/Clinical-Genomics/crunchy (to revision master) to ./tmp/pip-req-build-v3641sit
  Running command git clone --quiet https://github.com/Clinical-Genomics/crunchy /home/henrik.stranneheim/tmp/pip-req-build-v3641sit
  Resolved https://github.com/Clinical-Genomics/crunchy to commit a82b32b6818b1b1a1dd392b29c1924c9ae8507d6
  Preparing metadata (setup.py) ... done
Requirement already satisfied: click==8.1.3 in /home/proj/production/bin/miniconda3/envs/P_crunchy/lib/python3.10/site-packages (from crunchy==1.0.5) (8.1.3)
Requirement already satisfied: coloredlogs==15.0.1 in /home/proj/production/bin/miniconda3/envs/P_crunchy/lib/python3.10/site-packages (from crunchy==1.0.5) (15.0.1)
Requirement already satisfied: importlib-metadata==5.0.0 in /home/proj/production/bin/miniconda3/envs/P_crunchy/lib/python3.10/site-packages (from crunchy==1.0.5) (5.0.0)
Requirement already satisfied: humanfriendly==10.0 in /home/proj/production/bin/miniconda3/envs/P_crunchy/lib/python3.10/site-packages (from crunchy==1.0.5) (10.0)
Requirement already satisfied: zipp==3.9.0 in /home/proj/production/bin/miniconda3/envs/P_crunchy/lib/python3.10/site-packages (from crunchy==1.0.5) (3.9.0)
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/henrik.stranneheim
done.
Log deploy... done.
crunchy, version 1.0.5
[henrik.stranneheim@hasta:~] [P_base] $ conda run --name P_crunchy crunchy --help
Usage: crunchy [OPTIONS] COMMAND [ARGS]...

  Base command for crunchy

                .---. .---.
               :     : o   :    me want cookie!
           _..-:   o :     :-.._    /
       .-''  '  `---' `---' "   ``-.
     .'   "   '  "  .    "  . '  "  `.
    :   '.---.,,.,...,.,.,.,..---.  ' ;
    `. " `.                     .' " .'
     `.  '`.                   .' ' .'
      `.    `-._           _.-' "  .'  .----.
        `. "    '"--...--"'  . ' .'  .'  o   `.
        .'`-._'    " .     " _.-'`. :       o  :
      .'      ```--.....--'''    ' `:_ o       :
    .'    "     '         "     "   ; `.;";";";'
   ;         '       "       '     . ; .' ; ; ;
  ;     '         '       '   "    .'      .-'
  '  "     "   '      "           "    _.-'

Options:
  --spring-binary TEXT            Path to spring binary  [default: spring]
  --samtools-binary TEXT          Path to spring binary  [default: samtools]
  -t, --threads INTEGER           Number of threads to use for spring
                                  compression  [default: 8]
  --version                       Show the version and exit.
  -r, --reference TEXT            Path to reference genome
  --log-level [DEBUG|INFO|WARNING]
                                  Choose what log messages to show
  --tmp-dir TEXT                  If specific temp dir should be used
  --help                          Show this message and exit.

Commands:
  auto        Run whole pipeline by compressing, comparing and deleting...
  checksum    Generate the checksum for a file.
  compare     Compare two files by generating checksums.
  compress    Compress genomic files.
  decompress  Decompress genomic files

**Pull request merged**